integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -546,6 +548,30 @@ class CarbonSqlParser()
     colGrpNames.toString()
   }
 
+  protected def getTableBlockSize(tableProperties: Map[String, String]): Integer = {
+    var tableBlockSize: Integer = 0
+    if (tableProperties.get(CarbonCommonConstants.TABLE_BLOCKSIZE).isDefined) {
+      val blockSizeStr: String = tableProperties.get(CarbonCommonConstants.TABLE_BLOCKSIZE).get
+      try {
+        tableBlockSize = Integer.parseInt(blockSizeStr)
+      } catch {
+        case e: NumberFormatException =>
+          throw new MalformedCarbonCommandException("Invalid table_blocksize value found: " +
+            s"$blockSizeStr, only int value from 1 to 2048 is supported.")
+      }
+      if (tableBlockSize < CarbonCommonConstants.BLOCK_SIZE_MIN_VAL ||
+        tableBlockSize > CarbonCommonConstants.BLOCK_SIZE_MAX_VAL) {
+        throw new MalformedCarbonCommandException("Invalid table_blocksize value found: " +
+          s"$blockSizeStr, only int value from 1 to 2048 is supported.")
```

Comment:
Please mention the Unit also in comment like GB or MB
"Invalid table_blocksize value found: " +
 +          s"$blockSizeStr, only int value from 1 MB to 2048 MB is supported"